src/components/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -89,9 +105,10 @@ class UpgradePlanForm extends React.Component {
                 <FormattedMessage id="common.cancel" />
               </button>
               <button
+                type="submit"
```

Comment:
I will also apply this before merging:

![image](https://user-images.githubusercontent.com/1157864/54475725-c036d000-47d3-11e9-839c-8ed2e2660208.png)